openmdao/components/balance_comp.py

```diff
@@ -170,11 +170,12 @@ def apply_nonlinear(self, inputs, outputs, residuals):
 
```

Comment:
You are not finished with the story.  Saw this in the last PR too, so if you are pushing up early to get a good CI run in, we can talk to Steve about doing CI on your local branch, and also about fixing any problems in your local env.